web/src/app/admin/connectors/[connector]/AddConnectorPage.tsx

```diff
@@ -230,7 +230,7 @@ export default function AddConnector({
         }, 1000);
       }
     } else {
-      setPopup({ message: message, type: "error" });
+      setPopup({ message: message, type: isSuccess ? "success" : "error" });
```

Comment:
I would prefer the same message as the other connector creations + in the error case, we should probably show what the error is